Doc/using/cmdline.rst

```diff
@@ -994,3 +1000,110 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. _using-on-finding-modules:
+
+Finding modules
+---------------
+
+A module search path is initialized when Python starts. This module search path
+may be accessed at :data:`sys.path`.
```

Comment:
```suggestion
is accessed at :data:`sys.path`.
```

Reply:
`may be accessed` is correct IMO. The data is available at sys.path, but you don't have to go and view it, that is what I was getting at with `may be accessed`. The other correct alternative is `is available at sys.path`